packages/runtime/src/templates/server.ts

```diff
@@ -0,0 +1,104 @@
+import { PrerenderManifest } from 'next/dist/build'
+import { NodeRequestHandler, Options } from 'next/dist/server/next-server'
+
+import {
+  netlifyApiFetch,
+  getNextServer,
+  NextServerType,
+  normalizeRoute,
+  localizeRoute,
+  localizeDataRoute,
+  unlocalizeRoute,
+} from './handlerUtils'
+
+const NextServer: NextServerType = getNextServer()
+
+interface NetlifyConfig {
+  revalidateToken?: string
+}
+
+class NetlifyNextServer extends NextServer {
+  private netlifyConfig: NetlifyConfig
+  private netlifyPrerenderManifest: PrerenderManifest
```

Comment:
[dust] Opt for [private class members](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) instead of TypeScript's `private`.

```suggestion
  #netlifyConfig: NetlifyConfig
  #netlifyPrerenderManifest: PrerenderManifest
```

and then in the code, e.g. 

```javascript
    const manifest = this.getPrerenderManifest()
    this.#netlifyPrerenderManifest = {
      ...manifest,
      routes: { ...manifest.routes },
      dynamicRoutes: { ...manifest.dynamicRoutes },
    }
```

Reply:
as this is a subclass, i decided to follow the convention in the Next.js Server itself, so i think it makes sense to stick with the current syntax for consistency

Reply:
Gotcha. Thanks for the context!